docs/articles/blog/2017-6-7-testcafe-v0-16-0-released.md

```diff
@@ -0,0 +1,74 @@
+
+<!--more-->
+
+## Enhancements
+
+### ⚙ TypeScript support ([#408](https://github.com/DevExpress/testcafe/issues/408))
+
+In this release, we have added an ability to write tests in [TypeScript](https://www.typescriptlang.org/). Using of TypeScript brings you all the advantages of strongly typed languages: rich coding assistance, painless scalability, check-as-you-type code verification and much more.
+
+TestCafe bundles TypeScript declaration file with the npm package, so you have no need to install any additional packages.
+
+Just create a `.ts` file with the
+
+<!-- Use `js` instead of `ts` for this code block for proper code highlighting -->
+
+```js
+import { Selector } from 'testcafe';
+```
+
+and write your test.
+
+For details, see [TypeScript Support](https://devexpress.github.io/testcafe/documentation/test-api/typescript-support.html)
+
+### ⚙ Support running in Chrome in headless mode and in device emulator ([#1417](https://github.com/DevExpress/testcafe/issues/1417))
+
+Now TestCafe allows you to run your tests in Google Chrome in the headless mode and the device emulation mode.
+
+[Headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome) allows you to run tests in Chrome without any visible UI shell. To run tests in the headless mode use the `:headless` postfix:
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+Device emulation mode allows you to check how your tests works on mobile devices via Chrome's built-in [device emulator](https://developers.google.com/web/tools/chrome-devtools/device-mode/). To run tests in the device emulation mode specify `emulation:` and [device parameters](https://devexpress.github.io/testcafe/documentation/common-concepts/browser-support.md#available-chrome-options):
+
+```sh
+testcafe "chrome:emulation:device=iphone 6" tests/sample-fixture.js
+```
+
+For details, see [Using Chrome-specific Features](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/browser-support.html#using-chrome-specific-features).
+
+### ⚙ Support HTML5 Drag and Drop ([#897](https://github.com/DevExpress/testcafe/issues/897))
+
+Starting with this release, TestCafe supports HTML5 drag and drop, so you can test elements with the `draggable` [attribute](http://w3c.github.io/html/editing.html#the-draggable-attribute).
+
+### ⚙ Fixed URL for opening remote browsers ([#1476](https://github.com/DevExpress/testcafe/issues/1476))
+
+We have simplified the format of links that TestCafe generates when you [run tests on remote browsers](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/browser-support.html#browsers-on-remote-devices).
+
+Now, you have no need to type a unique link for each test run, all the links became constant. So, it is easier now to run tests on a remote device repeatedly: you can run them by navigating a link from your browser history.
+
+## Bug Fixes
```

Comment:
add https://github.com/DevExpress/testcafe-hammerhead/issues/1131 to the bug fixes